views/index.erb

```diff
@@ -44,16 +162,32 @@
   </div>
 <% end %>
 
-
+	
 <!-- Add your Javascript to this script tag -->
 <script language="javascript">
   console.log("I am Javascript!");
-
-  var profile_values = document.getElementsByClassName('profile-value');
-  var total_words = 0;
-
-  // TODO - Count the number of words in the profile.
-  // hint - look at what the innerHTML property provides.
-
-  document.getElementById('word-count').innerHTML = total_words;
+	
+//count all words in the 4 textareas i tried to do it by Class instead of ID but it did not work
+	
+ function countBoxWords(to){
+	 	var box1words = document.getElementById('txt1').value.split(' ').length-1;
+		var box2words = document.getElementById('txt2').value.split(' ').length-1;
+		var box3words = document.getElementById('txt3').value.split(' ').length-1;
+		var box4words = document.getElementById('txt4').value.split(' ').length-1;
```

Comment:
Hello Dan, initially  i tried to iterate with the loop but for some reason it wont work, i tried again this evening , entered a new parameter to the function so will locate the ID like this: onkeydown="countBoxWords('txt2','word-count');"
		 	onkeyup="countBoxWords('txt2','word-count');"	
		 	onmouseout="countBoxWords('txt2','word-count');"
and modified the function like this : 

function countBoxWords(from,to){
	 	var boxwords = document.getElementById(from).value.split(' ').length-1;
		var total_words = "";
		var i;
		for (i = 0; i < boxwords.length; i++) {
    total_words += boxwords[i];
}
		document.getElementById(to).innerHTML = total_words;
	}
I tested it and it does not work, i find it more confusing because of the value that needs to calculate based on the split(' ').length and the length that i gave to the actual i variable... if you could please tell me what am i doing wrong ? 
Many thanks,
Corina